README.md

```diff
@@ -18,10 +18,10 @@ installed and their bin folders have been added to `PATH`.
 
```

Comment:
We do need `analysis_options.yaml` (as they prefer now) for strong mode. There are still some errors to clean up. I'm just chugging along on the easy ones now.

Great to see testing on the package bots.

We could do travis. We'd just need to add the service account as a private ENVIRONMENT var. It's a bit involved (see [here](https://github.com/firebase/firebase-dart/blob/master/tool/travis.sh#L6)) and we can (should) only do it for non-pull requests. But it's possible